test/ctia/entity/target_record_test.clj

```diff
@@ -0,0 +1,101 @@
+(ns ctia.entity.target-record-test
+  (:require [clj-momo.test-helpers.core :as mth]
+            [clojure.test :refer [deftest is are join-fixtures testing use-fixtures]]
+            [ctia.entity.target-record :as target-record]
+            [ctia.properties :as p]
+            [ctia.test-helpers.aggregate :as aggregate]
+            [ctia.test-helpers.auth :as auth]
+            [ctia.test-helpers.core :as helpers]
+            [ctia.test-helpers.crud :refer [entity-crud-test]]
+            [ctia.test-helpers.fake-whoami-service :as whoami-helpers]
+            [ctia.test-helpers.field-selection :as field-selection]
+            [ctia.test-helpers.http :as http]
+            [ctia.test-helpers.pagination :as pagination]
+            [ctia.test-helpers.store :as store]
+            [ctim.examples.target-records :refer [new-target-record-minimal
+                                                  new-target-record-maximal]]))
+
+(use-fixtures :once (join-fixtures [mth/fixture-schema-validation
+                                    helpers/fixture-properties:clean
+                                    whoami-helpers/fixture-server]))
+
+(use-fixtures :each whoami-helpers/fixture-reset-state)
+
+(defn additional-tests [_ target-record-sample]
+  (testing "GET /ctia/target-record/search"
+    ;; only when ES store
+    (when (= "es" (p/get-in-global-properties [:ctia :store :target-record]))
```

Comment:
in tests, it tends to better to use when-not to when, so that one defaults to running the tests instead of skipping them

Else imagine the `[:ctia :store :target-record]` path was refactored

Reply:
I agree with you. I adopted this snippet from previous work without much thinking; tests for `identity_assertion` and `judgement` also use the same pattern. I'm not sure if I can remove this `when` altogether. @ereteog probably can give more insight on that.

Reply:
@vemv how would you rewrite that case? because here we only want it to run when the store is "es".

Reply:
> how would you rewrite that case? because here we only want it to run when the store is "es".

```clj
(when-not (#{"foo" "bar" "baz"} (p/get-in-global-properties [:ctia :store :target-record]))
  ...)
```

...of course this set could be `def`ed somewhere.

Reply:
I don't like this either tbh. Let me ship it as is for now. I promise you to try to make it nicer later.